sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -300,6 +311,18 @@ public static int getSessionTokenMismatchMaximumBackoffTimeInMs() {
             DEFAULT_SESSION_TOKEN_MISMATCH_MAXIMUM_BACKOFF_TIME_IN_MILLISECONDS);
     }
 
+    public static int getSpeculationType() {
+        return getJVMConfigAsInt(SPECULATION_TYPE, 0);
+    }
+
+    public static int speculationThreshold() {
+        return getJVMConfigAsInt(SPECULATION_THRESHOLD, 500);
```

Comment:
nit: we should define these default values in separate constant variables above like other methods.